Src/.editorconfig

```diff
@@ -205,48 +227,26 @@ dotnet_naming_style.begins_with_i.required_suffix =
 dotnet_naming_style.begins_with_i.word_separator = 
 dotnet_naming_style.begins_with_i.capitalization = pascal_case
 
-dotnet_diagnostic.CS1570.severity = silent
-dotnet_diagnostic.CS1572.severity = silent
-dotnet_diagnostic.CS1573.severity = silent
-dotnet_diagnostic.CS1574.severity = silent
-dotnet_diagnostic.CS1591.severity = silent
-dotnet_diagnostic.CS1712.severity = silent
-dotnet_diagnostic.CS1734.severity = silent
+# A method should not follow a class
+dotnet_diagnostic.SA1201.severity = None
 
-# SA1649: File name should match first type name
-dotnet_diagnostic.SA1649.severity = none
+# 'public' members should come before 'private' members
+dotnet_diagnostic.SA1202.severity = None
 
-file_header_template = Copyright (c) Microsoft. All rights reserved.\nLicensed under the MIT license. See LICENSE file in the project root for full license information.
-csharp_style_namespace_declarations = block_scoped:silent
-csharp_style_prefer_method_group_conversion = true:silent
-csharp_style_prefer_top_level_statements = true:silent
-csharp_style_prefer_null_check_over_type_check = true:suggestion
-csharp_style_prefer_local_over_anonymous_function = true:suggestion
-dotnet_diagnostic.SA1602.severity = suggestion
-dotnet_diagnostic.SA1307.severity = silent
+# Constant fields should appear before non-constant fields
+dotnet_diagnostic.SA1203.severity = None
 
-[*Tests.cs]
+# Constant fields should appear before non-constant fields
+dotnet_diagnostic.SA1203.severity = None
+
+# Static members should appear before non-static members
+dotnet_diagnostic.SA1204.severity = None
+
+# Readonly fields should appear before non-readonly fields
+dotnet_diagnostic.SA1214.severity = None
 
 # CS1591: Missing XML comment for publicly visible type or member
-dotnet_diagnostic.CS1591.severity = none
+dotnet_diagnostic.CS1591.severity = suggestion
 
```

Comment:
@schlaman-ms Wouldn't we want to enforce this?

Reply:
@schlaman-ms That's why we should discuss this standard _before_ making the change. If we do go with this standard, there should be spike on the implications, and only _then_ should we implement it and create a PR.

Reply:
Restored editorconfig back to original but added in necessary dropping of requirements for things we haven't decided on so we don't get warnings. Nothing added here just existing warnings taken away.